common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4120,6 +4120,12 @@
     ],
     "sqlState" : "07501"
   },
+  "NEW_CHECK_CONSTRAINT_VIOLATION" : {
+    "message" : [
+      "The new check constraint (<expression>) is violated by the existing data in the table <tableName>."
```

Comment:
I would reword to something like `The check constraint (<expression>) cannot be added because ...`.

Reply:
Sure, updated